frontend/src/router.tsx

```diff
@@ -53,14 +53,12 @@ const app_routes: RouteObject[] = [
   {
     path: Routes.Login,
     element: <Login />,
+    loader: async () => (await fetchAuthenticated()) && redirect("/"),
```

Comment:
Because the loader runs before the component is mounted. If we try to move this inside the Login component, it's going to render the login page, find out that it's already logged in, and redirect, which is not a great experience

Reply:
Mmmm interesting. Okay, let's add a comment about this

Reply:
Done